etc/HIProdOfflineConfiguration.py

```diff
@@ -114,7 +113,7 @@
 
 # Configure scenarios
 #ppScenario = "ppEra_Run3"
-ppScenario = "ppEra_Run3_2024_ppRef"
+ppScenario = "ppEra_Run3_2025_ppRef"
```

Comment:
We don't have Era_Run3_2025_ppRef in CMSSW.
https://github.com/cms-sw/cmssw/tree/master/Configuration/Eras/python